### PR TITLE
App icons

### PR DESCRIPTION
**Pull request by vderunov**

### CODE REVIEW

**Comment by vderunov** [comment]
photo_2024-03-15_01-49-08
Screenshot 2024-03-15 012250

**Comment by vderunov** [comment]
Screenshot 2024-03-15 142452

**Comment by vderunov** [comment]
Screenshot 2024-03-15 at 13 06 10

File: src/main.js [hunk 1]
```diff
@@ -82,7 +85,7 @@ function createWindow() {
     width: 600,
     height: 470,
     // frame: false,
-    icon: path.join(__dirname, '../../assets/icon.ico'),
+    icon: isDebug ? path.join(__dirname, '../../assets/icon.ico') : undefined,
```
Comment (noisekit) [inline src/main.js:88]:
```suggestion
    icon: isDebug ? path.join(app.getAppPath(), 'assets/icon.ico') : undefined,
```
just checking if we can use this here too?

**Review by noisekit — APPROVED** [review]
great!

File: src/main.js (hunk 1 above)
Comment (vderunov) [reply src/main.js]: I checked, it'is okay, I made the change, thanks for the comment.

**Pull request merged**